standalone/runtime/tests/test_issue.rs

```diff
@@ -629,9 +629,11 @@ fn integration_test_issue_wrapped_execute_liquidated() {
 
         assert_eq!(
             ParachainState::get(),
-            post_liquidation_status.with_changes(|user, _vault, liquidation_vault, fee_pool| {
+            post_liquidation_status.with_changes(|user, _vault, liquidation_vault, _fee_pool| {
                 user.free_tokens += issue.amount;
-                fee_pool.vault_rewards += issue.fee;
+
+                // since the vault has been liquidated execute rewards
+                // are transferred to the treasury
```

Comment:
Why did you make this change? Needs a spec change

Reply:
This was an implicit change since the Vault is liquidated before rewards are distributed.

Reply:
I don't see why that means you can't transfer the reward to fee pool, though?

Reply:
https://github.com/interlay/interbtc/blob/master/crates/reward/src/lib.rs#L197

Reply:
That condition should in practice never hold, though, right?

Reply:
Probably not, but it will likely hold in tests.

Reply:
That is not a reason to change the behavior - these tests just need to add some stake before calling. I'm not against the change in particular, especially since this is a rare edge case, but I think we shouldn't just change the behavior because it is more convenient for tests. If you want to have this change, then I think we should make it explicit in the spec, and/or have @nud3l sign off on this

Reply:
I think this is already included in the specification as far as I can see: https://spec.interlay.io/spec/fee.html#distributerewards

I think it would be good to just have an additional test with a second vault that actually has still stake so we know that the correct amount is distributed to the reward pool. What do you guys think?

Reply:
> I think this is already included in the specification as far as I can see

The [issue spec](https://github.com/interlay/interbtc-spec/blob/master/interbtc-spec/docs/source/spec/issue.rst?plain=1#L228) states it is transferred to the fee pool

Reply:
This is my concern with overlapping specifications, since as far as I was aware this change was already reflected in the fee component as highlighted there by @nud3l. In this case, the issue component is wrong and it should be updated but I'm not sure if there is a clearer path forward to link usage.

Reply:
In cases where lower-level functions are called, we should use something along the lines of: "Function [link] MUST return Ok". But good point, we should decide on a standard to not have these overlaps.